src/item.h

```diff
@@ -725,6 +725,14 @@ class item : public visitable<item>
          * @param mod How many charges should be removed.
          */
         void mod_charges( int mod );
+        /**
+         * Whether the item has to be removed as it has rotten away completely.
```

Comment:
You may want to add a note that this calls `item::process_temperature_rot` and thereby modifies the item. Currently it sounds as if this only query some condition.